app/models/katello/concerns/subscription_facet_host_extensions.rb

```diff
@@ -16,6 +16,8 @@ module SubscriptionFacetHostExtensions
 
         has_many :activation_keys, :through => :subscription_facet
         has_one :subscription_status_object, :class_name => 'Katello::SubscriptionStatus', :foreign_key => 'host_id'
+        has_one :hypervisor_host, :through => :subscription_facet
```

Comment:
i don't think this is needed anymore since you're using an ext_method for searching?